### ISSUE TITLE
Abstract with title not possible

### ISSUE THREAD

**Issue by mdreier** [asciidoctor/asciidoctor-pdf#698]
I was trying to create an abstract with a title in order to have different styles for the header and the remaining content. This is the relevant section of my custom style:

````
abstract:
  font_size: 18
  font_style: normal
  line_height: 1
  padding: [0.3cm, 0, 42, 0]
  title:
    align: left
    font_size: 20
    font_style: bold
````

(If you wonder about the large font size, I am using this as a workaround for #95).

When I use a non-structured abstract, the (non-title) abstract style is applied correctly.

````
= Title Lorem ipsum dolor sit amet
John Doe <john.doe@example.com>
v1.3, December 2016
:notitle:

[abstract]
*Headline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod* +
Subheadline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod tempor incididunt
````

However, as soon as I include a title in the abstract section, the abstract style is disregarded completely and the section is styled according to the normal heading and content styles.

````
= Title Lorem ipsum dolor sit amet
John Doe <john.doe@example.com>
v1.3, December 2016
:notitle:

[abstract]
== Headline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod
Subheadline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod tempor incididunt
````

Looking at the documentation it should be possible to have an abstract with a title. In case this is relevant, I am using the article document style.

I may be doing this incorrectly, but it looks more like an issue with the PDF generator to me.

**Comment by mojavelinux** [comment]
Currently, a section cannot act as an abstract in Asciidoctor (though it does seem to work by chance for the DocBook output). You would need to wrap the abstract in an open block and use a discrete heading as follows.

```
[abstract]
--
[discrete]
== Headline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod

Subheadline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod tempor incididunt
--
```

But what you likely want to do is just add an abstract title:

```
[abstract]
.Headline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod
Subheadline Lorem ipsum dolor sit amet, consectetaur adipisicing elit, sed doeiusmod tempor incididunt
```

The title for the abstract will be rendered in the soon to be released alpha.14. (See #582).

The first way you tried to write it seems reasonable to me, so I'll see about implementing that in core (and then in Asciidoctor PDF).

**Comment by mdreier** [comment]
I created my example from info given in the [asciidoctor user manual](http://asciidoctor.org/docs/user-manual/#user-abstract), so it may be a limitation only of the PDF part. In any case, you could add the information what is supported to the styling guide.

The first way you described does not work, is that also in the upcoming alpha.14?

**Comment by mojavelinux** [comment]
> The first way you described does not work, is that also in the upcoming alpha.14?

Yes.

> I created my example from info given in the asciidoctor user manual

Uh oh, it looks like that example is incorrect or the manual found a bug in the implementation. Which it is I'm not even sure yet. I'll file an issue in core to deal with it.

**Comment by mojavelinux** [comment]
Yes, that does appear to be a limitation in core after all. The AsciiDoc Python manual talks about abstract being a valid special section, so we should support it.
